.github/workflows/test.yml

```diff
@@ -252,3 +253,21 @@ jobs:
   #         ./contrib/localnet_liveness.sh 100 5 50 localhost
   #       if: env.GIT_DIFF
 
+
+  test-qgb-e2e:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.17
+      - uses: technote-space/get-diff-action@v6.0.1
+        with:
+          PATTERNS: |
+            **/**.go
+            go.mod
+            go.sum
+      - name: Test
+        run: go test -test.timeout 30m -failfast -v github.com/celestiaorg/celestia-app/e2e/qgb
```

Comment:
I want  to have the `-failfast` flag here so that if one test fails, the job doesn't keep running for more time and the dev can go run things locally and see what's the problem.
If not, we will need to wait for around 20 min for all the tests to execute,  which is too much.
What do you think?